### PR TITLE
Add SEM90 to vehicle racks

### PR DESCRIPTION
**Pull request by Soldia1138**
**When merged this pull request will:**
- Add SEM90 to vehicle racks

### CODE REVIEW

**Review by TheMagnetar — APPROVED** [review]
Looks good.

File: addons/sys_sem70/radio/fnc_getChannelData.sqf [hunk 1]
```diff
@@ -73,19 +73,22 @@ if (_manualChannel isEqualTo 1) then {
     HASH_SET(_return, "CTCSSRx", HASH_GET(_radioData, "CTCSS"));
     HASH_SET(_return, "modulation", HASH_GET(_radioData, "modulation"));
     HASH_SET(_return, "encryption", HASH_GET(_radioData, "encryption"));
-    HASH_SET(_return, "power", HASH_GET(_radioData, "power"));
     HASH_SET(_return, "squelch", HASH_GET(_radioData, "squelch"));
+    if (HASH_GET(_radioData, "powerSource") == "VAU") then {
+        HASH_SET(_return, "power", (HASH_GET(_radioData, "power") * 10));
```
Comment (TheMagnetar) [inline addons/sys_sem70/radio/fnc_getChannelData.sqf:78]: Should we change the multiplier to `GVAR()` or `#define` as well?

File: addons/sys_sem70/radio/fnc_getChannelData.sqf [hunk 2]
```diff
@@ -73,19 +73,22 @@ if (_manualChannel isEqualTo 1) then {
...
     HASH_SET(_return, "frequencyTX", HASH_GET(_channel, "frequencyTX"));
     HASH_SET(_return, "frequencyRX", HASH_GET(_channel, "frequencyRX"));
-    //HASH_SET(_return, "CTCSSTx", HASH_GET(_radioData, "CTCSS"));
-    //HASH_SET(_return, "CTCSSRx", HASH_GET(_radioData, "CTCSS"));
-    //HASH_SET(_return, "modulation", HASH_GET(_radioData, "modulation"));
-    //HASH_SET(_return, "encryption", HASH_GET(_radioData, "encryption"));
-    HASH_SET(_return, "power", HASH_GET(_radioData, "power"));
-    //HASH_SET(_return, "squelch", HASH_GET(_radioData, "squelch"));
+    if (HASH_GET(_radioData, "powerSource") == "VAU") then {
+        HASH_SET(_return, "power", (HASH_GET(_radioData, "power") * 10));
```
Comment (TheMagnetar) [inline addons/sys_sem70/radio/fnc_getChannelData.sqf:89]: Same as above.

File: addons/sys_sem70/radio/fnc_getChannelData.sqf (hunk 1 above)
Comment (Soldia1138) [reply addons/sys_sem70/radio/fnc_getChannelData.sqf:78]: Want to do it in a batch commit for all radios (on vehiclerack branch). With all the other Power `GVAR/#define` changes

Comment (TheMagnetar) [reply addons/sys_sem70/radio/fnc_getChannelData.sqf:78]: That is what I thought. 👍

**Pull request merged**